frontend/src/metabase/querying/fields/components/FieldPanel/utils.ts

```diff
@@ -43,7 +41,7 @@ function getGroupsWithColumns(
 
     return {
       columnItems,
-      displayName: groupInfo.displayName || t`Question`,
```

Comment:
AFAICT the goal of this PR is to not change any functionality and we're removing one `Question` label without replacing it with an equivalent on the back end.

It is not clear to me that this does not change functionality in this case.

Reply:
It replaces it with `Summaries` and makes things consistent.